device-types/Ivanti/PulseSecure-PSA300.yaml

```diff
@@ -0,0 +1,23 @@
+---
+manufacturer: Ivanti
+model: PulseSecure PSA300
+slug: ivanti-pulse-secure-psa300
+part_number: PSA300
+u_height: 1
```

Comment:
The PSA300 looks to not be a rack mounted device. If this is the case please mark this as `u_height: 0`

Reply:
Correct, I fixed it.
I had seen in other Device Types, which are also not rack mounted, that there is also u_height 1. I assumed that is a minimum height. But then I know for the future. Thanks.

Reply:
Unfortunately not, this is largely a community maintained repo so if you do notice any that need to be fixed please submit a PR! It would be much appreciated!